hw/ip/otp_ctrl/data/otp_ctrl_mmap.hjson

```diff
@@ -184,6 +184,46 @@
                     name: "CREATOR_SW_CFG_RMA_SPIN_CYCLES",
                     size: "4"
                 }
+                {
+                    name: "CREATOR_SW_RNG_REPCNT_THRESHOLDS",
+                    size: "4"
```

Comment:
For all these settings, I am allocating enough bits to set both boot and continuous entropy mode. ROM_EXT can then relax the continuous mode settings if needed.

Reply:
I assume these are two `uint16_t`s packed into a single `uint32_t`, correct?

Reply:
This is correct. The manufacturing script will have to be aware of this.